typescript/test/utils/mock-bridge.ts

```diff
@@ -20,6 +20,13 @@ interface RevealDepositLogEntry {
   deposit: Deposit
 }
 
+interface RedemptionBridgeLogEntry {
```

Comment:
Maybe just `RedemptionProofLogEntry`? This will be consistent with `DepositSweepProofLogEntry`

Reply:
Done in 75b561e1a724388f482970378e5a348d1ce35dbb.